builders/interoperability/xcm/xc-registration/assets.md

```diff
@@ -39,193 +39,107 @@ To create a forum post on the [Moonbeam Community Forum](https://forum.moonbeam.
 
 ### Create a Proposal to Register an Asset {: #create-a-proposal }
 
-To register an asset native to another chain on Moonbeam, you'll need to submit a governance proposal that will call the `assetManager.registerForeignAsset` extrinsic. Additionally, you'll need to set the asset's units per second value through the `assetManager.setAssetUnitsPerSecond` extrinsic. The units per second value is the units of tokens to charge per second of execution time during XCM transfers. How to calculate the units per second will be covered in the following section.
+To register an asset native to another chain on Moonbeam, you'll need to submit a governance proposal that will call the `evmForeignAssets.createForeignAsset` extrinsic. Additionally, you'll need to register the asset's price (formerly known as units per second) through the `xcmWeightTrader.addAsset` extrinsic so that XCM fees can be properly calculated. An existing asset's price can be updated with `xcmWeightTrader.editAsset`. 
 
 To get started, you'll need to collect some information about the asset:
 
-- The ID of the parachain the asset lives on
-- The metadata of the asset. This includes:
-  - The asset name
-  - The asset symbol. You'll need to prepend "xc" to the asset symbol to indicate that the asset is an XCM-enabled asset
-  - The number of decimals the asset has
-  - The units per second
-- The multilocation of the asset as seen from Moonbeam
+- `assetId`: unique identifier of the asset
+- `xcmLocation`: a scale encoded xcm v4 multilocation of asset reserve relative to Moonbeam
+- The asset name
+- The asset symbol. You'll need to prepend "xc" to the asset symbol to indicate that the asset is an XCM-enabled asset
+- The number of decimals the asset has
 
-With this information in hand, you can get the encoded calldata for both calls and batch the calldata into a single transaction. From there, you can start the governance process, which includes using the calldata to submit a preimage and then using the preimage to create a proposal. If you're also opening a channel at the same time, you can add the channel-related calldata to the batch asset registration calldata and open a single proposal for everything. Asset and channel registration proposals on Moonbeam should be assigned to the General Admin Track.
+With this information in hand, you can prepare a governance proposal to register a foreign asset. Foreign asset registration proposals should be submitted through the `FastGeneralAdmin` track.
```

Comment:
Asset  registration needs to be done via General Admin.

THe only thing that can be done in FastGeneralAdmin is opening the channel -> https://github.com/moonbeam-foundation/moonbeam/blob/master/runtime/moonbeam/src/xcm_config.rs#L641

From Moonbase -> https://github.com/moonbeam-foundation/moonbeam/blob/527a930e6f02522edc26047c36518b3b712705cf/runtime/moonbase/src/xcm_config.rs#L699-L705